config/tsan.suppressions

```diff
@@ -10,3 +10,5 @@ race:apache::thrift::server
 thread:libcuda.so
 # 2020-02-23: resolve thread safety issues when acquiring string dict proxy through resultset 
 race:ResultSet::makeTargetValue 
+
+race:^tbb*
```

Comment:
Is this coming from the same place as the arrow race previously?

Reply:
Not really. As we found, building tbb with TSAN and ignoring race:rml::* (which is tbb internal resource manager) also takes warnings away. So, we can assume that there are no races in user code. 
The only warning remains is arrow::read_csv which is of cause false positive, as arrow is well tested with tsan enabled on their side. 
So, it seems, there are no problems on our side, but we need to somehow disable TSAN warnings.